File: backend/src/main/java/com/example/backend/member/dto/MemberResponse.java
package com.example.backend.member.dto;

import com.example.backend.member.entity.Member;
import com.example.backend.member.entity.Role;
import lombok.Builder;
import lombok.Getter;

@Getter
@Builder
public class MemberResponse {

    private Long id;
    private String email;
    private String name;
    private String phone;
    private Role role;
    private String roleDescription;
    private String department;
    private String licenseNumber;

    public static MemberResponse from(Member member) {
        return MemberResponse.builder()
                .id(member.getId())
                .email(member.getEmail())
                .name(member.getName())
                .phone(member.getPhone())
                .role(member.getRole())
                .roleDescription(member.getRole().getDescription())
                .department(member.getDepartment())
                .licenseNumber(member.getLicenseNumber())
                .build();
    }
}

File: Project/Models/MemberResponse.cs
using System.Text.Json.Serialization;

namespace Project.Models
{
    public class MemberResponse
    {
        [JsonPropertyName("id")]
        public long Id { get; set; }

        [JsonPropertyName("email")]
        public string Email { get; set; } = string.Empty;

        [JsonPropertyName("name")]
        public string Name { get; set; } = string.Empty;

        [JsonPropertyName("phone")]
        public string? Phone { get; set; }

        [JsonPropertyName("role")]
        public string Role { get; set; } = string.Empty;

        [JsonPropertyName("roleDescription")]
        public string? RoleDescription { get; set; }

        [JsonPropertyName("department")]
        public string? Department { get; set; }

        [JsonPropertyName("licenseNumber")]
        public string? LicenseNumber { get; set; }
    }
}
